airbyte-integrations/connectors/source-zendesk-talk/source_zendesk_talk/schemas/ivrs.json

```diff
@@ -49,7 +49,7 @@
                   "type": ["null", "object"]
                 },
                 "overflow_options": {
-                  "type": ["null", "object"]
+                  "type": ["null", "array"]
```

Comment:
do we know the type of array elements?

Reply:
unfortunately, we don't, I wasn't able to populate it and docs says just array, response from the server is an empty array, so SAT was failing here